### ISSUE TITLE
[NewPlugin] Community Buff Notifier

### ISSUE THREAD

**Issue by tristo7** [Ishadijcks/MooLite#77]
**What part of the game does your Plugin relate to?**
Most/All skills

**What should your feature do?**
A Notification when a community buff is added/expires

**What kind of input and output does your feature need?**
configuration around which buffs trigger the notification

**Comment by Ishadijcks** [comment]
Good idea, I haven't done anything regarding community buffs yet, so we would first need to capture and reflect that data

**Comment by Ishadijcks** [comment]
Depends on #78
